documentatie/use-cases/GSB-gebruikersdoelen.md

```diff
@@ -107,33 +107,29 @@ __uitbreidingen__:
 - OSV richt de server ook gelijk als client in. Willen wij dit ook?
 - Afzenderverificatie ontbreekt nog, want nog geen beslissing over oplossing.
 
-## GSB richt de applicatie in
+
+## De coördinator maakt de stembureaus aan.
 
 __niveau:__ gebruikersdoel, zee, 🌊
 
 ### Hoofdscenario en uitbreidingen
 
```

Comment:
Hebben we hier nog precondities nodig?
- een lijst met stembureaus hebben (afkomstig uit een andere applicatie waarin stembureaus worden beheerd, of een export uit Abacus van een eerdere verkiezing)

Reply:
Twee precondities: EML-NL bestand om te importeren en de gepubliceerde lijst om mee te controleren.